check_python.sh

```diff
@@ -52,14 +52,21 @@ if [ -n "${project_check_module}" ]; then
   cm_pylint_out="${PWD}/pylint.log"
   rm -f "${cm_pylint_out}"
   # Must suppress issues being written to stdout.
-    "${project_check_module}" > check_module.out
-    if [ -s "${cm_pylint_out}" ]; then
+  "${project_check_module}" > check_module.out
+  rc=$?
```

Comment:
I prefer `${PIPESTATUS[0]}` as it's future-proof to somebody adding `| <some_filter>` to the command without noticing they are altering the checking of the status on `${project_check_module}`.
```suggestion
  rc=${PIPESTATUS[0]}
```